website/blog/2022/04/17/md.md

```diff
@@ -0,0 +1,313 @@
+
+    # TYPE apisix_batch_process_entries gauge
+
+    apisix_batch_process_entries{name="http logger",route_id="1",server_addr="172.19.0.8"} 0
+
+    # HELP apisix_etcd_modify_indexes Etcd modify index for APISIX keys
+
+    # TYPE apisix_etcd_modify_indexes gauge
+
+    apisix_etcd_modify_indexes{key="consumers"} 17819
+
+    apisix_etcd_modify_indexes{key="global_rules"} 17832
+
+    apisix_etcd_modify_indexes{key="max_modify_index"} 20028
+
+    apisix_etcd_modify_indexes{key="prev_index"} 18963
+
+    apisix_etcd_modify_indexes{key="protos"} 0
+
+    apisix_etcd_modify_indexes{key="routes"} 20028
+
+    ...
+
+```
+
+And we can also check the status of our endpoint at the Prometheus dashboard by pointing to this URL `http://localhost:9090/targets`
+
+[![plugin-orchestration-configure-rule-screenshot](https://res.cloudinary.com/practicaldev/image/fetch/s--3oVnBx6W--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/jg8mlc7tifrduomwp5vj.png)](https://res.cloudinary.com/practicaldev/image/fetch/s--3oVnBx6W--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/jg8mlc7tifrduomwp5vj.png)
+
+As you can see, Apache APISIX exposed metrics endpoint is upon and running.
+
+Now you can query metrics for `apisix_http_status` to see what HTTP requests are handled by API Gateway and what was the outcome.
+
+[![prometheus-plugin-dashboard-query-http-status-screenshot](https://res.cloudinary.com/practicaldev/image/fetch/s--t7CroAIQ--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/36qcbqggieet4wms0ivu.png)](https://res.cloudinary.com/practicaldev/image/fetch/s--t7CroAIQ--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/36qcbqggieet4wms0ivu.png)
+
+[![prometheus-plugin-dashboard-query-http-status-table-screenshot](https://res.cloudinary.com/practicaldev/image/fetch/s--owBpTjKI--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/d7jyr8xo5uny70vcb4rd.png)](https://res.cloudinary.com/practicaldev/image/fetch/s--owBpTjKI--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/d7jyr8xo5uny70vcb4rd.png)
+
+In addition to this, you can view the Grafana dashboard running in your local instance. Go to `http://localhost:3000/`
+
+[![prometheus-plugin-grafana-dashboard-screenshot](https://res.cloudinary.com/practicaldev/image/fetch/s--4-6fmBbW--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/mdpwdm84lqmbd9cyytt0.png)](https://res.cloudinary.com/practicaldev/image/fetch/s--4-6fmBbW--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/mdpwdm84lqmbd9cyytt0.png)
+
+You can also check two other plugins for metrics:
```

Comment:
Here are two Tracers links? `skywalking` and `opentelemetry` only be used by Tracers. I think it should be `prometheus`  here.